community/kernel/src/main/java/org/neo4j/kernel/impl/index/labelscan/NativeLabelScanStore.java

```diff
@@ -99,7 +96,7 @@
     /**
      * Native label index tag, to distinguish native label index from other label indexes
      */
-    public static final String NATIVE_LABEL_INDEX_TAG = GraphDatabaseSettings.LabelIndex.NATIVE.name();
+    public static final String NATIVE_LABEL_INDEX_TAG = "native";
```

Comment:
To not depend on deprecated enum. Do you think it is better to wait with this change until the enum is actually removed?